content/_index.md

```diff
@@ -3,14 +3,25 @@ title: Introduction
 type: docs
 ---
 
-# SCRC docs 
+# SCRC 
 
-Greetings people of the SCRC. Welcome to our documentation website. I hope you find it useful!
+## Who are we?
+The Scottish COVID-19 Response Consortium is a consortium originally formed by three organisations
 
-If you want to edit or add to any of the pages here, please click on the "Edit this page" link at the bottom of the page you want to edit. This will take you to a GitHub repository, which stores the source files for that particular page. From there you can click on the tiny pencil (on hover: Edit this file), which will allow you to make any edits you want. Please remember that any tables or fancy formatting will have to be written in markdown / html to be visible. If you need any help with this please [get in touch](mailto:sonia.mitchell@glasgow.ac.uk). Once you're finished, just click the link at the bottom of the page to create a new branch and start a pull request.
+- The [Boyd Orr Centre for Population and Ecosystem Health](https://www.gla.ac.uk/research/az/boydorr/) at the University of Glasgow
+- The Scottish Government's [Centre of Expertise on Animal Disease Outbreaks](https://www.epicscotland.org/) (EPIC)
+- [Biomathematics and Statistics Scotland](https://www.bioss.ac.uk/)
```

Comment:
I'm not sure there's any reason to single us out any longer, particularly on this site. I think we can either just say something like "formed by dozens of individuals from over 30 academic and commercial organisations" and link to the university at https://www.gla.ac.uk/research/az/scrc/ourpeople/#members , or (if you're really keen) list all of the organisations, starting with those three. I'd definitely go for the former though!

Reply:
Thanks - gone for the former. It's be great, I think, to get a list of people currently involved on this site so we can see who has what skills / who's doing what. Maybe I should raise an issue against this repo?